README.md

```diff
@@ -33,16 +33,99 @@ BlockScout supports a number of projects. Hosted instances include POA Network,
 - [List of hosted mainnets, testnets, and additional chains using BlockScout](https://docs.blockscout.com/for-projects/supported-projects)
 - [Hosted instance versions](https://docs.blockscout.com/about/use-cases/hosted-blockscout)
 
-
 ## Getting Started
 
+1. Install requirements
+
+    For a complete list of requirements, see the [blockscout docs](https://docs.blockscout.com/for-developers/information-and-settings/requirements).
+     > Note that we use older versions of Elixir and Erlang (see `.tool-versions`).  For help installing and managing these versions, you can follow the instructions in this article [here](https://medium.com/juq/how-to-manage-elixir-versions-on-mac-or-linux-getting-started-with-elixir-12308e7b6451).
+     >
+     > If it is still not recognizing Elixir, you can try adding this to your `~/.zshrc` file: `PATH=/$HOME/asdf/installs/elixir/1.9.1/.bin:$PATH`.  You can find your correct path using `asdf which elixir`.
+
+2. Set up some default configuration
+
+    ```shell
+    cp apps/explorer/config/dev.secret.exs.example apps/explorer/config/dev.secret.exs
+    cp apps/block_scout_web/config/dev.secret.exs.example apps/block_scout_web/config/dev.secret.exs
+    ```
+
+3. Add Env Variables
+
+4. Install Deps and Compile
+
+    ```shell
+    mix local.hex --force
+    mix local.rebar --force
+    mix deps.get
+    cd apps/block_scout_web/assets/ && \
+      npm install && \
+      npm run build && \
+      cd -
+    cd apps/explorer/ && \
+      npm install && \
+      cd -
+    mix compile
+    ```
+
+5. Start blockscout in a docker container
+
+     > If you do not already have docker installed, you can get it here:  
+     >   - [Mac](https://hub.docker.com/editions/community/docker-ce-desktop-mac)
+     >   - [Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows)
+
+    ```shell
+    cd docker
+    make start
```

Comment:
Something like
```suggestion
NETWORK=Celo ETHEREUM_JSONRPC_VARIANT=geth ETHEREUM_JSONRPC_HTTP_URL=http://104.198.100.15:8545 ETHEREUM_JSONRPC_WS_URL=ws://104.198.100.15:8546 COIN=cGLD make start
```

should take care of everything including creating the database container, running migrations and starting indexer, so steps 6-9 are not needed really. Maybe it worth mentioning that values for 
`ETHEREUM_JSONRPC_HTTP_URL` and `ETHEREUM_JSONRPC_WS_URL` may vary and should point to a running archive node.